src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,219 @@
+                                          (not horizontal-description?))
+                                 [rn/image {:source {:uri mid-section-description-user-icon}
+                                            :style  {:width             32
+                                                     :height            32
+                                                     :border-radius     32
+                                                     :margin-horizontal 6}}])
+                               [rn/view {:style {:flex-direction (if horizontal-description?
+                                                                   :row
+                                                                   :column)}}
+                                [rn/text {:size   :base
+                                          :weight :font-semi-bold
+                                          :style  {:color      text-color
+                                                   :text-align :left}}
+                                 mid-section-main-text]
+                                (when mid-section-description
+                                  [rn/view {:style {:flex-direction :row}}
+                                   (when (and mid-section-description-icon
+                                              (not horizontal-description?))
+                                     [icons/icon mid-section-description-icon
+                                      (icon-props text-secondary-color :small)])
+                                   [rn/text {:size   :small
+                                             :weight :font-medium
+                                             :style  {:padding-right 4
+                                                      :margin-left   2
+                                                      :text-align    :left
+                                                      :line-height   18
+                                                      :color         text-secondary-color}}
+                                    mid-section-description]])]])]))
+
+(defn right-section-icon
+  [props {:keys [bg icon icon-color]}]
+  [rn/view {:style (merge
+                    icon-styles
+                    {:background-color bg
+                     :width            32
+                     :height           32}
+                    props)}
+   [icons/icon icon (icon-props icon-color :big)]])
+
+(defn page-nav
+  [{:keys [one-icon-align-left? horizontal-description? align-mid page-nav-color page-nav-background-uri mid-section-type mid-section-icon mid-section-main-text mid-section-left-icon
+           mid-section-right-icon mid-section-description mid-section-description-color mid-section-description-icon mid-section-description-user-icon
+           mid-section-main-text-icon-color left-section-icon left-section-icon-color
+           left-section-icon-bg-color right-section-icons]
+    :or   {align-mid                         false
```

Comment:
im not sure if we really need all these dummy default values here, better to pass them from quo screens ,

Reply:
Added